pkg/codegen/tmpl/core_resource.tmpl

```diff
@@ -1,6 +1,29 @@
 package {{ .PackageName }}
 
-// Resource is the wire representation of {{ .KindName }}. (TODO be better)
+import (
+	"github.com/grafana/grafana/pkg/kinds"
+)
+
+// Resource is the kubernetes style representation of {{ .KindName }}. (TODO be better)
```

Comment:
I'm gonna start interpreting "move to app sdk" as "move to app sdk or kindsys" and just not worry about it :)

Yes, this will need to move, and it needs uniformity here and elsewhere.